tests/src/Kernel/Access/OgEntityAccessTest.php

```diff
@@ -133,6 +177,15 @@ protected function setUp() {
       ->grantPermission($this->randomMachineName())
       ->save();
 
+    $this->ogAdminRole = OgRole::create();
+    $this->ogAdminRole
+      ->setId($this->randomMachineName())
+      ->setLabel($this->randomString())
+      ->setGroupType($this->group1->getEntityTypeId())
+      ->setGroupBundle($this->groupBundle)
+      ->setIsAdmin(TRUE)
+      ->save();
```

Comment:
Ok, in front of a computer, so here's the reference `\Drupal\og\GroupManager::createPerBundleRoles`. That is, `\Drupal\og\GroupManager::getDefaultRoles` should allow having roles that are `is_admin`